dev-java/jdbc-mysql/jdbc-mysql-8.3.0.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source test"
+MAVEN_ID="com.mysql:mysql-connector-j:8.3.0"
+# Tests disabled, see bug #902723
+# JAVA_TESTING_FRAMEWORKS="junit-4"
+
+inherit java-pkg-2 java-pkg-simple
+
+# Bundling a binary version of oci-java-sdk-common
+# https://github.com/oracle/oci-java-sdk/blob/v2.47.0/bmc-common/pom.xml
+OSC="3.29.0"
+
+DESCRIPTION="JDBC Type 4 driver for MySQL"
+HOMEPAGE="https://dev.mysql.com/doc/connector-j/en/"
+SRC_URI="https://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-j-${PV}.tar.gz
+	https://repo1.maven.org/maven2/com/oracle/oci/sdk/oci-java-sdk-common/${OSC}/oci-java-sdk-common-${OSC}.jar"
+
+S="${WORKDIR}/mysql-connector-j-${PV}"
+
+LICENSE="GPL-2-with-MySQL-FLOSS-exception"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+
+CP_DEPEND="
+	dev-java/c3p0:0
+	=dev-java/protobuf-java-3.25.3-r0:0
```

Comment:
does it really need just this specific version or the condition can be relaxed a bit? like `3.25*` for example.

Reply:
> does it really need just this specific version or the condition can be relaxed a bit? like `3.25*` for example.

that's exactly what i was looking for. just updated.